[2022-02-25 01:58:09,764] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Project_Mapping.load_to_s3 manual__2022-02-25T01:53:04.902631+00:00 [queued]>
[2022-02-25 01:58:09,810] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Project_Mapping.load_to_s3 manual__2022-02-25T01:53:04.902631+00:00 [queued]>
[2022-02-25 01:58:09,813] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-25 01:58:09,815] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-25 01:58:09,817] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-25 01:58:09,871] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): load_to_s3> on 2022-02-25 01:53:04.902631+00:00
[2022-02-25 01:58:09,880] {standard_task_runner.py:52} INFO - Started process 3698 to run task
[2022-02-25 01:58:09,890] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Project_Mapping', 'load_to_s3', 'manual__2022-02-25T01:53:04.902631+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/mydag.py', '--cfg-path', '/tmp/tmp5nj676q5', '--error-file', '/tmp/tmpye5rvhi9']
[2022-02-25 01:58:09,893] {standard_task_runner.py:77} INFO - Job 65: Subtask load_to_s3
[2022-02-25 01:58:10,127] {logging_mixin.py:109} INFO - Running <TaskInstance: Project_Mapping.load_to_s3 manual__2022-02-25T01:53:04.902631+00:00 [running]> on host 717d92cd82bb
[2022-02-25 01:58:10,385] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Project_Mapping
AIRFLOW_CTX_TASK_ID=load_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-02-25T01:53:04.902631+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-25T01:53:04.902631+00:00
[2022-02-25 01:58:49,074] {logging_mixin.py:109} INFO - Negative
[2022-02-25 01:58:50,693] {python.py:175} INFO - Done. Returned value was: None
[2022-02-25 01:58:50,749] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Project_Mapping, task_id=load_to_s3, execution_date=20220225T015304, start_date=20220225T015809, end_date=20220225T015850
[2022-02-25 01:58:50,911] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-25 01:58:51,015] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
